### PR TITLE
multi: propagate zero conf fields to the `ChannelAcceptResponse`

### PR DESCRIPTION
**Pull request by positiveblue**
Make lndclient zeroconf ready

#### Pull Request Checklist

- [X] PR is opened against correct version branch.
- [ ] Version compatibility matrix in the README and minimal required version
      in `lnd_services.go` are updated. <= only after merging to master?

### CODE REVIEW

**Review by guggero — APPROVED** [review]
LGTM :tada:

**Comment by guggero** [comment]
> only after merging to master?

We just need to check that the minimum version of this branch was set to `v0.15.1` (maybe a previous PR missed it?).

**Comment by guggero** [comment]
Can be rebased.

File: lightning_client.go [hunk 1]
```diff
@@ -1025,6 +1035,11 @@ type AcceptorResponse struct {
 	// MinAcceptDepth is the number of confirmations we require before we
 	// consider the channel open.
 	MinAcceptDepth uint32
+
+	// Zeroconf signals that the responder wants this to be a zero-conf
```
Comment (guggero) [inline lightning_client.go:1039]: nit: match capitalization of field name to make it a proper Godoc comment.

**Review by guggero — APPROVED** [review]
LGTM :tada:

**Comment by positiveblue** [comment]
@guggero I updated the lnd minimum version, I think we forgot to do it when we created the branch

**Review by guggero — APPROVED** [review]

**Pull request merged**